src/main/java/javax/validation/ConstraintValidatorContext.java

```diff
@@ -567,9 +567,9 @@ ContainerElementNodeBuilderCustomizableContext addContainerElementNode(
 
 		/**
 		 * Represents refinement choices for a node which is
-		 * in an {@code Iterator} or {@code Map}.
+		 * in an iterable, e.g. array, {@code Iterable} or {@code Map}.
 		 * <p>
-		 * If the iterator is an indexed collection or a map,
+		 * If the iterable is an indexed collection or a map,
```

Comment:
You change it to `iterable` here and to `container` in the change just below. Shouldn't we be consistent?